docs/sources/reference/api.md

```diff
@@ -592,6 +592,16 @@ JSON post body can be sent in the following format:
 }
 ```
 
+You can optionally attach [non-indexed labels]({{< relref "../get-started/labels/non-indexed-labels" >}}) to each log line by adding a JSON object to the end of the log line array.
+The JSON object must be a valid JSON object with string keys and string values.
+The JSON object must be set immediately after the log line. Here is an example of a log entry with some non-indexed labels attached:
```

Comment:
Can we clarify that the JSON shouldn't have nested keys?

Reply:
Modified with:
_The JSON object must be a valid JSON object with string keys and string values. **The JSON object should not contain any nested object**._